xc7/tests/srl/srl16_amc31/CMakeLists.txt

```diff
@@ -0,0 +1,46 @@
+add_file_target(FILE basys3_doutmux_top.v SCANNER_TYPE verilog)
+#add_file_target(FILE basys3_dffmux_top.v SCANNER_TYPE verilog)
```

Comment:
Why are the dffmux tests commented out?

Reply:
That is because I cannot verify it on hardware. The VPR refuses to use FF on the same slice as the SRL. I'll try to play more with pack patterns, maybe I can get it to do so.

Reply:
@litghost I discovered that in order to have such a pack pattern we would need to make different COMMON_SLICE and SLICE_FF pb_types for SLICEL and SLICEM. That is because MC31 related pack patterns cannot be specified inside SLICEL as this makes VPR report an error.

I made specialized COMMON_SLICE and SLICE_FF but still couldn't force VPR to pack SRL and FF in the same slice.

I think that having the DFFMUX.MC31 working is not a top priority. I can file an issue with a description what (supposedly) needs to be done.

Reply:
> I think that having the DFFMUX.MC31 working is not a top priority. I can file an issue with a description what (supposedly) needs to be done.

This makes sense to me for now.

Reply:
I added comments to CMake files. There is the issue: https://github.com/SymbiFlow/symbiflow-arch-defs/issues/974